project_national/sdr_upgrades_tmy3_full_coverage.yml

```diff
@@ -0,0 +1,1176 @@
+schema_version: '0.4'
```

Comment:
Have you thought about creating this file based on `sdr_upgrades_tmy3.yml`? Seems pretty risky/inconvenient to have to maintain 2 different versions of the same yml file.

Reply:
Oh, for sure. I created this new sdr_upgrades_tmy3_full_coverage just for testing purpose since the sdr_upgrades_tmy3.yml is being used in existing integration test with 2 data points only. I will eventually migrate the sdr integration test to this one and get rid of the sdr_upgrades_tmy3_full_coverage version.